### ISSUE TITLE
Glob issues

### ISSUE THREAD

**Issue by joelbourbon** [nolemmings/compare-json#10]
Your example does not seem to be working on Windows. I have to explicitly list all the files.

I feel that this works only on Linux where the command line would, by itself, expand the glob to a list of files.

Am I right?
Is there any way you could fix that?

**Comment by nielskrijger** [comment]
Hi Joel,

This package uses https://www.npmjs.com/package/glob to expand globs, it shouldn't depend on any linux inputs.

I do see some comments about windows within that glob-package README, can you see if those apply? It says you must use "/" instead of "\\" on windows (which surprised me tbh).

If that doesn't solve the problem, can you give me the exact command that is failing? I.e. is it:

```
comparejson ./test/fixture/*.json
```
exactly?

I do have a windows machine but no dev tools installed on it at all, so it would take me some effort to debug :-)

**Comment by joelbourbon** [comment]
Hi @nielskrijger,

Thanks for your prompt reply!

After reading [this documentation](https://www.npmjs.com/package/glob#windows), I still don't understand what is wrong with the way I am trying to use it.

My exact command is:
```
comparejson -e ./src/resources/*.json
```
So I am not using any "\" (backslash) but only "/" (forward slash).

Looking at your code rapidly, I see you do have a dependency on `glob` [but you only seem to be using it in your tests](https://github.com/nolemmings/compare-json/search?utf8=%E2%9C%93&q=glob&type=).

Once again, thanks a lot for your support!

**Comment by joelbourbon** [comment]
My guess is that `glob` should be used right [here](https://github.com/nolemmings/compare-json/blob/master/lib/cli.js#L28) to expand `files`.

**Comment by nielskrijger** [comment]
@joelbourbon you were right, I installed it on my windows machine and got the same behaviour.

I've released a new version 0.4.0. Can you check if that works for you? If not feel free to reopen.

**Comment by joelbourbon** [comment]
Hi @nielskrijger,

Works like a charm!

Thanks for the fast support!
